src/components/data_search/DatasetSearchTableDisplay.tsx

```diff
@@ -62,13 +63,16 @@ export const DatasetSearchTableDisplay = (props: DatasetSearchTableDisplayProps)
     </Box>
   )
     : (
-      <SimpleTable
-        rowData={rowData}
-        columnHeaders={headers}
-        selected={selected}
-        styles={styles}
-        tableSize={10}
-        summary='faceted dataset search table'
-      />
+      <>
+        <div style={{ fontWeight: 600, borderBottom: '1px solid black' }}>{rowData.length} {_.capitalize(rowData.length > 1 ? tab.plural : tab.singular)}</div>
```

Comment:
An interesting interaction is that if you filter to `0` records, the `isEmpty(filteredData)` condition kicks in and this block, along with the table, disappears. I think it would be cleaner to reflect that condition on this line, but it certainly isn't required. In that case, if we did show `0` records, the plural would be the correct language to use here.

Reply:
I agree; I was expecting that the total would be outside the table, so it would be displayed in all cases. I don't see a specific UX proposal on the ticket so I may have missed a discussion about the UX in refinement.

Instead of showing a total inside the table, I think it would be better to show the total at the top, maybe on the same lines as the search (filter) text entry box. Also I would include both the number shown and the total number, so a user can tell how much of the total they're omitting. Text like "Showing X out of Y"/ "Showing all of Y"/"Showing none of Y" is one way to format this information.

Reply:
I think for now my inclination is to leave the present `isEmpty` behavior, because it reduces redundant info. If necessary we can update it later.